### ISSUE TITLE
Change the format of printed Full .NET Framework Version

### ISSUE THREAD

**Issue by adamsitnik** [dotnet/BenchmarkDotNet#1111]
As of today BDN prints something like this:

```log
DefaultJob : .NET Framework 4.7.2 (CLR 4.0.30319.42000), 64bit RyuJIT-v4.8.3752.0
```

We have three numbers here: 4.7*, 4.0* and 4.8*.

I took a look at the code and the numbers represent the following:

```
.NET Framework {InstalledDotNetSdKVersion} (CLR {Environment.Version}), 64bit {ProcessModules.JitModuleVersion}
```

The `Environment.Version` does not provide any useful data, I  think that we should simplify the output to the following:

```
.NET Framework SDK 4.7.2 (CLR v4.8.3752.0), 64bit RyuJIT
```

@AndreyAkinshin do you agree?

/cc @billwert

**Comment by AndreyAkinshin** [comment]
@adamsitnik 
1. Do we really need the "SDK" postfix in ".NET Framework SDK"? What about computers without installed SDK (InProcessToolchain should work fine there)
2. Are you sure that it's correct to write "CLR v4.8.3752"? Currently, all .NET Framework 4.x uses that same folder: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\`

**Comment by billwert** [comment]
For #2, The install directory will never change again. Using the version off of clrjit.dll or clr.dll is much more informative.

Thanks,
-Bill
________________________________

**Comment by adamsitnik** [comment]
Fixed by #1114
